modules/flowable-variable-service/src/main/java/org/flowable/variable/service/impl/persistence/entity/HistoricVariableInstanceEntityManagerImpl.java

```diff
@@ -146,12 +143,10 @@ public List<HistoricVariableInstanceEntity> findHistoricalVariableInstancesBySub
 
     @Override
     public void deleteHistoricVariableInstancesByTaskId(String taskId) {
-        if (serviceConfiguration.isHistoryLevelAtLeast(HistoryLevel.ACTIVITY)) {
```

Comment:
Given that the `if` is removed  shouldn't the indentation of the following lines be changed too?